File: product/src/app/model/order.ts
export class Order {

    //Table ID
    id?: number = 0;

    //Customer Data
    customerId: number = 0;
    customerName?: string = "";
    orderId: number = 0;
    phoneNumber: string = "";

    //Product Data
    productId: number = 0;
    productName: string = "";
    imageFile: string = "";
    quantity: number = 0;
    price: string = "0.0";
    uom: string = "";
    address: string = ""; 
    //SKU??

    //status
    status: string ="";

    //transaction Data
    transactionId: number = 0;
    transactionDate: string ="";
    totalAmount: string = "";
   // paymentType: string = ""; //not yet implemented in orderlist/database comment for now
}

File: ecommerce/src/main/java/com/narvaez/model/Order.java
package com.narvaez.model;

import lombok.Data;

import java.util.Date;

@Data
public class Order {
    int id;
    int orderId;
    int customerId;
    String customerName;

    //Product Data
    int productId;
    String productName;
    String imageFile;
    double quantity;
    String uom;
    double price;
    String status;
    String Address;

    //TRANSACTION DATA
    String transactionId;
    Date transactionDate;
    double totalAmount;
}
